quickstart/llmd-infra-installer.sh

```diff
@@ -293,11 +293,6 @@ uninstall() {
     $KCMD delete crd servicemonitors.monitoring.coreos.com --ignore-not-found || true
   fi
 
-  if [[ "${USE_MINIKUBE}" == "true" || "$(kubectl config current-context 2>/dev/null)" == "minikube" ]]; then
-    log_info "🗑️ Minikube context found; deleting hostPath PV (${MODEL_PV_NAME})"
-    kubectl delete pv "${MODEL_PV_NAME}" --ignore-not-found || true
-  fi
-
   log_info "🗑️ Deleting ClusterRoleBinding llm-d"
   $KCMD delete clusterrolebinding -l app.kubernetes.io/instance=llm-d
```

Comment:
I think we can remove these too, we dont install any CRBs in infra

Reply:
@Gregory-Pereira done 👍